test/example.test.js

```diff
@@ -0,0 +1,16 @@
+// IMPORT MODULES under test here:
```

Comment:
having an empty test file is what is causing you to fail travis. Having a testing suite and no tests and a red flag for travis.